go.sum

```diff
@@ -1,6 +1,17 @@
 github.com/awesome-gocui/termbox-go v0.0.0-20190427202837-c0aef3d18bcc h1:wGNpKcHU8Aadr9yOzsT3GEsFLS7HQu8HxQIomnekqf0=
```

Comment:
Maybe just delete and re-create this file so we don't have all the termbox lines in here

Reply:
fixed in 7e477c7